luigi/parameter.py

```diff
@@ -139,7 +159,10 @@ def __init__(self, default=_no_value, is_global=False, significant=True, descrip
                                 ``positional=False`` for abstract base classes and similar cases.
         :param bool always_in_help: For the --help option in the command line
                                     parsing. Set true to always show in --help.
-        :param str batch_method: How multiple
+        :param BatchAggregation batch_method: How multiple values of this argument are combined when
+                                              batching in the scheduler. See BatchAggregation for
+                                              details of what each one means. Default is None,
+                                              meaning that this parameter cannot be aggregated.
```

Comment:
This is good docs, but for this big new feature I expect there to be something similar to what we have for [docs for Range*](https://github.com/spotify/luigi/blob/master/doc/luigi_patterns.rst#triggering-recurring-tasks). Why not add it in the same file `luigi_patterns.rst`? In particular make sure to explain these cases:

 * It should be clear to the user what happens when you have some batched and some non-batched params.
 * It should be clear what happens when you have an IntParameter but batch as COMMA_LIST. (that's the reason I don't like COMMA_LIST, can't we have a python list instead?)